charm_test.go

```diff
@@ -4,27 +4,33 @@
 package charm_test
 
 import (
+	"bufio"
 	"bytes"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	stdtesting "testing"
+	"strings"
 
+	"github.com/juju/testing"
 	jc "github.com/juju/testing/checkers"
 	"github.com/juju/utils/fs"
 	gc "gopkg.in/check.v1"
+	"gopkg.in/juju/charm.v6"
 	"gopkg.in/yaml.v2"
 
-	"gopkg.in/juju/charm.v6"
+	coretesting "github.com/juju/juju/testing"
```

Comment:
This charm.v6 repo must not depend on juju/juju

Reply:
Ok. Iam using IsolationSuite from github.com/juju/testing